x-pack/plugins/spaces/server/lib/saved_objects_client/spaces_saved_objects_client.ts

```diff
@@ -211,4 +212,22 @@ export class SpacesSavedObjectsClient implements SavedObjectsClientContract {
       namespace: spaceIdToNamespace(this.spaceId),
     });
   }
+
+  /**
+   * Updates an array of objects by id
+   *
+   * @param {array} objects - an array ids, or an array of objects containing id, type, attributes and optionally version, references and namespace
+   * @returns {promise} - { saved_objects: [{ id, type, version, attributes }] }
+   * @example
+   *
+   * bulkUpdate([
+   *   { id: 'one', type: 'config', attributes: { title: 'My new title'}, version: 'd7rhfk47d=' },
+   *   { id: 'foo', type: 'index-pattern', attributes: {} }
+   * ])
+   */
+  public async bulkUpdate(objects: SavedObjectsBulkUpdateObject[] = []) {
```

Comment:
We should maintain the same signature as the base client by exposing `options` here, and then `throwErrorIfNamespaceSpecified(options)`. I know that `namespace` is the only option currently, but it'll be one less obscure bug to hunt down if/when options is ever augmented.

Reply:
Dammit, I should have asked, as I contemplated that point and landed on the side of making it impossible to express the invalid state of having a namespace there at all.

But yeah, for consistency's sake I can definitely see that as a reasonable option.

Reply:
Done